lib/utils.js

```diff
@@ -669,3 +671,64 @@ function autoEnd (stream, eventName) {
     stream.flushSync()
   }
 }
+
+/**
+ * Returns a object which will be used to handle custom levels
+ *
+ * @param {string|object} cLevels Custom Levels can be object or string
+ *
+ * @returns {object} A object like level: level_name
+ */
+function handleCustomlevelsOpts (cLevels) {
+  if (!cLevels) return {}
+
+  if (typeof cLevels === 'string') {
+    return cLevels
+      .split(',')
+      .reduce((agg, value, idx) => {
+        const [levelName, levelIdx = idx] = value.split(':')
+        agg[levelIdx] = levelName.toUpperCase()
+        return agg
+      },
+      { default: 'USERLVL' })
+  } else if (typeof cLevels === 'object') {
+    return Object
+      .keys(cLevels)
+      .reduce((agg, levelName, idx) => {
+        agg[cLevels[levelName]] = levelName.toUpperCase()
+        return agg
+      }, { default: 'USERLVL' })
+  } else {
+    return {}
+  }
+}
+
+/**
+ * Returns a object which will be used to handle custom levels
+ *
+ * @param {string|object} cLevels Custom Levels can be object or string
+ *
+ * @returns {object} A object like level_name: level
+ */
```

Comment:
Please adjust this docblock in the same manner as my suggestion on the other one.

Reply:
Ok